draft-ietf-mls-protocol.md

```diff
@@ -549,6 +537,62 @@ X, then the tree will have the following structure.
 X    B          C    D
 ~~~~~
 
+A node secret is derived from its path secret. The path secret
+for the parent is derived from the same path secret (from the children)
+both using the KDF defined below, but with different labels.
+The private and public encryption keys for a node are derived from the
+node secret using the Derive-Key-Pair. These computations are as follows:
+
+~~~~~
+path_secret[n+1]
+     ^
+     |
+path_secret[n] --> node_secret --> node_key_pair
+~~~~~
+
+~~~~~
+node_secret = HKDF-Expand-Label(path_secret[n], "node", "", Hash.Length)
+node_private, node_public = Derive-Key-Pair(node_secret)
+path_secret[n+1] = HKDF-Expand-Label(path_secret[n], "path", "", Hash.Length)
+~~~~~
+
+where Derive-Key-Pair can be defined as a function where given
```

Comment:
Everything from here down should be deleted, and replaced with the existing definitions that have been deleted in this PR.